### PR TITLE
Base files for boot modules

### PR DESCRIPTION
**Pull request by PuKoren**
Contains the boot modules loaded upon module start.

See README for more info:
https://github.com/dial-once/node-microservice-boot/tree/feature/basic-boot-scripts

With this module we should be able to just `require('@dialonce/boot')` on our microservices and it will configure common things (env vars, loggers, bugsnag init, etc.).

It have retro-compatibility on env vars (`LOGENTRIES_TOKEN`, `BUGSNAG_TOKEN`) but should be initiated like this:

```js
require('@dialonce/boot')({
    LOGS_TOKEN: '',
    BUGS_TOKEN: ''
});
```

This way, we will be able to change the underlying bug/log reporter and just rebuild each module, without having to change everything because env var name don't match the service name).

What do you think about it @mrister @jkernech @ky23 ?

### CODE REVIEW

File: test/health-spec.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+var assert = require('assert'),
+  health = require('../src/health');
+
+describe('health', function() {
```
Comment (jkernech) [inline test/health-spec.js]: Sometimes `function()` is used, sometimes the ES6 syntax is used. Is there a specific reason for that ?

Comment (PuKoren) [inline test/health-spec.js]: No reason here, usually I use `function` to avoid using parent scope in tests. I will clean this up

Comment (PuKoren) [inline test/health-spec.js]: @jkernech done

**Comment by jkernech** [comment]
LGTM

**Comment by ky23** [comment]
@PuKoren Thanks, We don't have do worry about the boot anymore, just use this one.

**Pull request merged**